2-mandatory/2-oxygen-levels.js

```diff
@@ -11,7 +11,16 @@
     Some string methods that might help you here are .replace() and .substring().
 */
 
-function findSafeOxygenLevel() {}
+function findSafeOxygenLevel(array) {
+  let found = array.find((e) => {
+    if (Number(e.slice(0, -1)) !== NaN && e.slice(-1) === "%") {
```

Comment:
According to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NaN what are the correct ways to check if a value is actually `NaN`?